backend/internal/models/user.go

```diff
@@ -0,0 +1,12 @@
+package models
+
+/*
+User model for PostGres backend
+*/
```

Comment:
1. C-style comment syntax isn't necessary for just one line
2. This model isn't specific to Postgres at all, it's intended to be the standard representation of a user across the entire backend.
3. The G in Postgres is lowercase :)

It would be much more useful to explain that this represents a end user of the app (a family, not an admin), and that the ID is the UID of an account in Firebase Auth